ee/codegen/src/generators/workflow-sandbox-file.ts

```diff
@@ -0,0 +1,102 @@
+import { python } from "@fern-api/python-ast";
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+import {
+  ChatHistoryInput,
+  JsonInput,
+  NumberInput,
+  StringInput,
+} from "vellum-ai/api";
+
+import { vellumValue } from "src/codegen";
+import { BasePersistedFile } from "src/generators/base-persisted-file";
+import { getGeneratedInputsModulePath } from "src/utils/paths";
+
+type SandboxInput = StringInput | JsonInput | ChatHistoryInput | NumberInput;
+export type SandboxInputs = SandboxInput[];
```

Comment:
If this type is being imported into other files, let's move out of `generators/**` and somewhere in to `types/**`